manual/src/refman/modtypes.etex

```diff
@@ -200,8 +200,8 @@ For specifying a module component that is a functor, one may write
 instead of
 \begin{center}
 @'module' module-name ':'
- 'functor' '(' name_1 ':' module-type_1 ')' '->' \ldots
-                                            '->' module-type@
+  '(' name_1 ':' module-type_1 ')' '->' \ldots
```

Comment:
Missing arrow?
```suggestion
  '(' name_1 ':' module-type_1 ')' '->' \ldots '->'
```

Reply:
(Or remove the arrow before `\ldots`.)

Good catch!

Reply:
> (Or remove the arrow before `\ldots`.)

I picked this option (26beb2cf).